backend/globaleaks/handlers/whistleblower/submission.py

```diff
@@ -160,10 +166,35 @@ def db_create_receivertip(session, receiver, internaltip, tip_key):
     session.add(receivertip)
     return receivertip
 
+def is_valid_uuid(value):
+    try:
+        return str(uuid.UUID(value, version=4)) == value
+    except ValueError:
+        return False
```

Comment:
NOTE: code like this should be moved into globaleaks/utils/utility.py (where for example are already present  some utility functions releated to uuid) and it could be created a unit test globaleaks/tests/utils/test_utility.py  to raise at once possibilities of  code reuse and code coverage.